up42/catalog.py

```diff
@@ -503,7 +503,7 @@ def download_quicklooks(
                 with open(out_path, "wb") as dst:
                     for chunk in response:
                         dst.write(chunk)
-            except ValueError:
+            except IOError:
```

Comment:
It was getting before ValueError due to the bug fixed in auth.py in this PR - err.response.json() was failing due to invalid response content